include/seqan/basic/allocator_interface.h

```diff
@@ -204,6 +204,7 @@ allocate(T const &,
     if (data)
         SEQAN_PROADD(SEQAN_PROMEMORY, count * sizeof(TValue));
 #endif
+	memset((void*)data, 0, count * sizeof(TValue));
```

Comment:
yes very scarry :)
test_sequence failed. I could narrow it down to seqan::string's internal memory not being initialized correctly (I compared the internal data after creation in vs2015 and gcc, it was different). After I added this memset, the unit test was ok for vs2015 and the internal data was the same between gcc and vs2015.
I did not understand all the internals completely, so probably only the designer can say if this memset should be neccessary or not. There is the possibility that a compiler bug in vs2015 makes this memset neccessary. But I think its more likely, that gcc and clang set the allocated memory to zero - even if they dont have to.